Example/src/main/java/com/parse/example/Room.java

```diff
@@ -0,0 +1,27 @@
+package com.parse.example;
+
+import com.parse.ParseClassName;
+import com.parse.ParseObject;
+
+@ParseClassName("Room")
+public class Room extends ParseObject {
+
+    public Room() {
+
+    }
+
+    String name;
+
+    public String getName() {
+        return name;
+    }
+
+    public void setName(String name) {
+        this.name = name;
+    }
```

Comment:
Same issue in [Message.java](https://github.com/ParsePlatform/ParseLiveQuery-Android/pull/16/files#diff-f2c4c51701f223c290f207771bfc1249) as well.